README.md

```diff
@@ -1,70 +1,262 @@
+## <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/539px-React-icon.svg.png" width="23" height="20"/> React Frontend <a name="react-frontend"></a>
 
-## Learn More
+- Backend part of the project [Fast-Lane-Racer-frontend](https://github.com/muneeb3123/Fast-Lane-Racers-front-end)
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## 🚀 Live Demo <a name="live-demo"></a>
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+- [Live Demo]()
 
-### Code Splitting
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+## 💻 Getting Started <a name="getting-started"></a>
 
-### Analyzing the Bundle Size
+To get a local copy up and running, follow these steps.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+### Prerequisites
 
-### Making a Progressive Web App
+To get a local copy up and running follow these simple example steps.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+### Prerequisites
 
-### Advanced Configuration
+In order to run this project you need:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+- Code Editor (Vs Code)
+- Terminal
+- Install Ruby on your computer
+- Install rails on your computer
+- Node.js
+- PostgreSQL
+
+### Setup
+
+Clone these repositories by using the command line to your desired folder:
+
+<u>**For Backend**</u>
+
+```sh
+git clone https://github.com/muneeb3123/Fast-Lane-Racers-back-end.git
+
+cd Fast-Lane-Racer-backend
+```
+
+<u>**For Frontend**</u>
+
+```sh
+git clone https://github.com/muneeb3123/Fast-Lane-Racers-front-end.git
+
+cd Fast-Lane-Racer-frontend
+```
+
+### Install
+
+Install this project with:
+
+<u>**For Backend**</u>
+
+```sh
+  bundle install
+```
+
+<u>**For Frontend**</u>
+
+```sh
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+<u>**For Backend**</u>
+
+```sh
+  Set up JWT secret key as follow:-
+  rails secret (copy secret_key)
+
+  Linux/Mac user:-
+  export DEVISE_JWT_SECRET_KEY=secret_key
+
+  Window user:-
+  set DEVISE_JWT_SECRET_KEY=secret_key
+  or
+  $env:DEVISE_JWT_SECRET_KEY="secret_key"
+
+  rails server
+```
+
+<u>**For Frontend**</u>
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  Backend:-  rspec
+  Frontend:- npm test
```

Comment:
- [ ] You are not including frontend tests:

![image](https://github.com/muneeb3123/Fast-Lane-Racers-front-end/assets/57421823/f9990b20-f887-40d9-ba68-d5c1399a272a)

Kindly, either include some or remove the instructions, in order to prevent confusion. 👍